weblate/trans/models/component.py

```diff
@@ -718,6 +728,7 @@ def save(self, *args, **kwargs):
                 or (old.repo != self.repo)
                 or (old.branch != self.branch)
                 or (old.filemask != self.filemask)
+                or (old.screenshot_filemask != self.screenshot_filemask)
```

Comment:
```suggestion
```

This can be omitted, we do not need to perform a full repo scan on screenshot file mask change.